docs/content/python/guides/enforcement/field.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Field-level Enforcement"
+weight: 30
+any: true
+description: >
+  Learn about enforcing field-level authorization, controlling who can access
+  which fields of your resources.
+# draft: True
+---
+
+{{% callout "Note: 0.20.0 Alpha Feature" %}}
+  This is an API provided by the alpha release of Oso 0.20.0, meaning that it is
+  not yet officially released. You may find other docs that conflict with the
+  guidance here, so proceed at your own risk! If you have any questions, don't
+  hesitate to [reach out to us on Slack](https://join-slack.osohq.com). We're
+  here to help.
+{{% /callout %}}
+
+<div class="pb-10"></div>
+
+# Field-level Enforcement
+
+Perhaps you're creating a user profile page, and in your app, only a user's
+friends can access their last check-in location. To enforce this type of
+authorization, it often makes sense to use **"field-level" enforcement**, by
+explicitly allowing access to certain fields on your domain objects.
+
+Field-level authorization gives you fine-grained control over who can access
+exactly what bits of information. In polar, you can write field-level rules like
+this:
+
+```polar
+allow_field(user, "read", profile, "last_check_in_location") if
```

Comment:
I think we should include specializers on most rules in our documentation examples since it's best practice.